### ISSUE TITLE
Output JSON properties that are not ALLCAPS.

### ISSUE THREAD

**Issue by dbarros** [Rudolph-Miller/jonathan#68]
Is it possible to configure _jonathan_ so that it can output, and parse, JSON that has properties that are not in ALLCAPS?

For example, the JSON may use all lowercase for property name, or camelCase, or snake_case. e.g.:

`{
    "first_name": "Charlie";
  }` 

or 

`{
    "firstName": "Charlie";
  }`

**Comment by jg20019** [comment]
Yes. One way you can do it is to when you output json, you use write-key-value and with-object directly. There is an example on this page: http://rudolph-miller.github.io/jonathan/. You can find it in the example, where you define a method to-json to output a user class as json

**Comment by jg20019** [comment]
I found a better way:
    
    (jonathan:to-json '(("first_name" . "charlie")) :from :alist)
